docs/reference/selfhosted.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Self Hosted Infra
+position: 5
+---
+
+In most ways, self-hosted offers similar features to our Software as a Service offering. In fact, the main difference is that you host the "Server" on your own Kubernetes cluster. This document will show you how to install that server. Since the rest of the product is the same in the SaaS model and Self Hosted, you can refer to the rest of the documentation for everything else.
```

Comment:
Add a header title here so it renders better since the title isn't automatically rendered

```
# Self Hosted Infra
```

Reply:
yup